### PR TITLE
story/BODS-362: Update to the organisations data catalogue csv

### PR DESCRIPTION
**Pull request by ndemenezes**

### CODE REVIEW

File: transit_odp/organisation/csv/organisation.py [hunk 1]
```diff
@@ -624,6 +670,11 @@ def _get_organisation_details_dataframe() -> DataFrame:
         .values(*ORG_USER_FIELDS)
     )
 
+    if is_fares_validator_active:
+        orgs["archived_matching_report_url"] = orgs.apply(
+            lambda x: get_archived_matching_report_url(x), axis=1
+        )
+
```
Comment (surabhika) [inline transit_odp/organisation/csv/organisation.py:677]: I think you should put this part of the code inside `_prepare_calculated_columns()` function.

**Review by surabhika — CHANGES_REQUESTED** [review]

File: transit_odp/organisation/csv/organisation.py (hunk 1 above)
Comment (ndemenezes) [reply transit_odp/organisation/csv/organisation.py]: @surabhika I've pushed this change.

Comment (surabhika) [reply transit_odp/organisation/csv/organisation.py]: Could you please increment the version to get this into dev?

Comment (ndemenezes) [reply transit_odp/organisation/csv/organisation.py]: @surabhika Incremented the version number

Comment (surabhika) [reply transit_odp/organisation/csv/organisation.py]: I don't think you've pushed it?

Comment (ndemenezes) [reply transit_odp/organisation/csv/organisation.py]: yhh it wasn't showing up for some reason, it should have come through now

**Review by surabhika — APPROVED** [review]

**Pull request merged**